go/cmd/vttestserver/main.go

```diff
@@ -137,6 +137,9 @@ func init() {
 
 	flag.StringVar(&config.Charset, "charset", "utf8mb4", "MySQL charset")
 	flag.StringVar(&config.Collation, "collation", "", "MySQL collation")
+
+	flag.StringVar(&config.PlannerVersion, "planner_version", "v3", "Sets the default planner to use when the session has not changed it. Valid values are: V3, Gen4, Gen4Greedy and Gen4Fallback. Gen4Fallback tries the new gen4 planner and falls back to the V3 planner if the gen4 fails. All Gen4 versions should be considered experimental!")
```

Comment:
Since this is in main/13.0, should we still say Gen4 is experimental?

Reply:
Gen4 is still marked experimental on main and in the planner_version help of vtgate. I replicated it here. When we make that the default, we should also do it for this.